src/mds/Server.cc

```diff
@@ -472,6 +472,14 @@ void Server::handle_client_reclaim(const cref_t<MClientReclaim> &m)
     return;
   }
 
+  fs_cluster_id_t fsid = mds->get_fsid();
+  if (fsid != FS_CLUSTER_ID_NONE && !session->fsid_capable(fsid, MAY_READ)) {
+    dout(0) << " dropping message not allowed for this fsid: " << *m << dendl;
+    // TODO: talked with doug, not sending message anymore.
+    // m->put();
```

Comment:
This line led to a build error. `MClientReclaim` is indirectly inherited from `RefCountedObject` with public inheritance but `SafeMessage` makes the methods `get` and `put` private. I talked with Doug on this, he said it's OK to drop the message.

Reply:
yes, remove it.